### ISSUE TITLE
Australian Scheme Feedback: (p15) Scheme Guidance (non editorial)

### ISSUE THREAD

**Issue by jgb1128** [CC-in-the-Cloud/Admin#175]
"Additionally, guidance for penetration testing should be given with the understanding that the TOE may not be deployed in a fully closed environment and as such there may be limitations on the rules of engagement that must be followed with the CSP. It is expected that evaluation schemes will provide guidance on the penetration test efforts that will be accepted for cloud evaluations, and PP
authors should incorporate such guidance into PPs for consistency."

_It isn’t clear what types of penetration testing are being scoped out.  Is it traffic floods causing a DDoS and impacting the CSP’s other customers? Regardless, malicious actors don’t follow the CSP’s limited rules of engagement for penetration testing.
Should that section also say that, for types of penetration testing which the CSP disallows, an evaluation of the underlying CSP should explain/demonstrate/prove how the CSP defends against such types of penetration testing and similar actions performed by malicious actors?_

**Comment by tstodart** [comment]
Yes, could add that text in the context of 'may' rather than 'should'. AVA effort/difficulty is always in the context of the AVA_VAN level claimed (e.g. AVA_VAN.1 only public search).

**Comment by bharveyTX** [comment]
Change implemented, new paragraph:

Additionally, guidance for penetration testing may be given with the understanding that the TOE may not be deployed in a fully closed environment. As such, there may be limitations on the rules of engagement that must be followed with the CSP. It is expected that evaluation schemes will provide guidance on the penetration test efforts that will be accepted for cloud evaluations, and PP authors should incorporate such guidance into PPs for consistency.
